ui/imports/Style.qml

```diff
@@ -10,11 +11,10 @@ QtObject {
 
     property var changeTheme: function (theme) {
         switch (theme) {
-            case "light": current = lightTheme; break;
-            case "dark": current = darkTheme; break;
+            case Universal.Light: current = lightTheme; break;
+            case Universal.Dark: current = darkTheme; break;
             default: current = lightTheme; console.log('Unknown theme. Valid themes are "light" and "dark"')
```

Comment:
Something very important to notice here:

There has never been an option to actually: Set the theme to dark, or light, if system is dark or light respectively.
While we do have enum variants `Dark`, `Light` and `System` that can be used to tell our app what theme to use, there doesn't seem to be a way to actually determine **what** `System` is.

Therefore, so far we've always defaulted to `Light`.

In other words, there's no such thing as "default to `System`". I assume there's a way to determine what the current configured theme on the OS level is inside Qt/Nim, but there's certainly no way in QML and I couldn't find anything about it either.

Reply:
Aaaah, and I thought it was a Windows bug. So it actually never worked even on Mac 🤦